src/common/utils/removeDuplicate.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This function remove duplicates string value of an array.
+ * It removes also null, undefined or non string values.
+ *
+ * @param {array} array Array of values.
+ * @ignore
+ */
+const removeDuplicate = (array) => {
```

Comment:
For readability, please do this directly in plase.

Reply:
... or the mixin

Reply:
If you explain me how, without duplicating code, I do it.

Reply:
fixed